fw/tls.c

```diff
@@ -1014,7 +1025,7 @@ tfw_tls_cfgend(void)
 static int
 tfw_tls_start(void)
 {
-	tfw_tls.allow_any_sni = allow_any_sni_reconfig;
+	tfw_tls_allow_any_sni = allow_any_sni_reconfig;
```

Comment:
Now we have `tfw_tls_match_any_sni_to_dflt` that name do not match to implemented behaviour, I propose to rename it.